spec.html

```diff
@@ -41755,6 +41832,25 @@ <h1>Properties of the Map Constructor</h1>
         <li>has the following properties:</li>
       </ul>
 
+      <emu-clause id="sec-map.groupby">
+        <h1>Map.groupBy ( _items_, _callbackfn_ )</h1>
+        <emu-note>
+          <p>_callbackfn_ should be a function that accepts two arguments. `groupBy` calls _callbackfn_ once for each element in _items_, in ascending order, and constructs a new Map of arrays. Each value returned by _callbackfn_ is used as a key in the Map. For each such key, the result Map has an entry whose key is that key and whose value is an array containing all the elements for which _callbackfn_ returned that key.</p>
```

Comment:
```suggestion
          <p>_callbackfn_ should be a function that accepts two arguments. `groupBy` calls _callbackfn_ once for each element in _items_, in ascending order, and constructs a new Map. Each value returned by _callbackfn_ is used as a key in the Map. For each such key, the result Map has an entry whose key is that key and whose value is an array containing all the elements for which _callbackfn_ returned that key.</p>
```

(@syg suggested this change earlier, but I think it got lost)

Reply:
I'm confused, usually "a Map of X" means that X describes the values; is it just that it's somewhat redundant with the more verbose explanation later?

Reply:
Alrighty, updated.